config/crd/bases/cosmos.strange.love_cosmosfullnodes.yaml

```diff
@@ -555,6 +555,20 @@ spec:
                   Example: cosmos-1
                   Used for debugging.
                 type: object
+              ordinal:
```

Comment:
This should be `ordinals`

PS. There are two descriptions.
- ordinals
- start

Please copy these from above comments.

Reply:
Thanks for this catch , the json tag needed to be corrected